node/src/main.rs

```diff
@@ -320,13 +320,18 @@ async fn main() {
             );
             graph::spawn_blocking(job_runner.start());
         }
+        let restart_on_dynamic_data_source =
+            !env::var_os("EXPERIMENTAL_DISABLE_RESTART_ON_DYNAMIC_DS")
+                .map(|_| true)
+                .unwrap_or(false);
```

Comment:
What do you think of trying to give this change a more meaningful and shorter name. Perhaps "static filters"? This would mean renaming `restart_on_dynamic_data_source` to `static_filters_enabled` and flipping the conditionals since enabling static filters means restarts are disabled.

Reply:
sounds good!